src/lxml/html/clean.py

```diff
@@ -207,7 +207,7 @@ class Cleaner(object):
     remove_tags = None
     allow_tags = None
     kill_tags = None
-    remove_unknown_tags = True
+    remove_unknown_tags = None
```

Comment:
This change seems unnecessary, and it obfuscates the default configuration.
```suggestion
    remove_unknown_tags = True
```